### PR TITLE
Fixed some issues

### PR DESCRIPTION
**Pull request by kaiPushOrigin**
Closed issues #19 and #23. Also added the description for sharing links on fb.

### CODE REVIEW

**Comment by valencik** [comment]
Can you add the image file to the repo as well?

File: index.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
     <meta content="noindex" name="robots">
+    <meta property="og:image" content="http://cs.smu.ca/~ka_kashif/SMU-Hack-A-Week-1/images/HackAWeektwitter.jpg">
```
Comment (Frozenfire92) [inline index.html]: Should this be a link to the proper username that is serving the website?

Comment (kashifkai28) [inline index.html]: Yes I forgot to change that. I'll do that soon.

**Comment by kashifkai28** [comment]
@valencik : The image is already in the repo. Its in "images/HackAWeektwitter.jpg". We need to give the full URL though in og:image.
@Frozenfire92 : This should work now. Thanks :)

**Comment by valencik** [comment]
@kashifkai28 I thought you were going to create a new square image for this?
Have you tested this image with Facebook? Does it look as expected?

If it looks alright then that's fine.

**Comment by kashifkai28** [comment]
This is what it looks like - 

screen shot 2015-04-26 at 10 27 13 pm

**Comment by valencik** [comment]
So the cut off is not idea, and this defintiely doesn't work as an avatar (issue #19).
Could you create a new image asset suitable for an avatar and Facebook usage.
Ideally 1-1 aspect ratio and including the black outline seen in the official banner (http://cs.smu.ca/hackaweek/).

**Comment by kashifkai28** [comment]
Does this work? @valencik 

screen shot 2015-04-26 at 11 32 14 pm

**Comment by kashifkai28** [comment]
Does this work? @valencik 
screen shot 2015-05-03 at 8 35 06 pm

**Comment by valencik** [comment]
Thanks!

**Pull request merged**